static/bidder-params/andbeyondmedia.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Compass Adapter Params",
```

Comment:
Is this an alias of the Compass adapter? If so, we can advise on better ways to create the alias.

Reply:
@SyntaxNode hello, this is not an alias, this is a separate adapter